packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/ExtractionService.java

```diff
@@ -73,12 +73,16 @@ public class ExtractionService {
 	private final CurrentUserService currentUserService;
 
 	// time the progress takes to reach each subsequent half
-	final Double HALFTIME_SECONDS = 2.0;
+	final Double HALFTIME_SECONDS = 5.0;
+
+	// Allow up to 10 concurrent extraction tasks
+	// TODO: Move this to application properties or set the POOL_SIZE in a smarter way
+	private final int POOL_SIZE = 10;
```

Comment:
Can we move this to `application.properties` now before this becomes a foreverToDo :D?

Reply:
@dvince2 could you do it on this PR, I don't think @jryu01 is confident enough right now to do it properly, and I do not want this PR to drag for long.

Reply:
done and done!